bin.src/gen2to3.py

```diff
@@ -39,13 +39,15 @@ def main(config=None):
     convertRepoConfig.skyMaps["discrete/ci_hsc"] = ConvertRepoSkyMapConfig()
     convertRepoConfig.skyMaps["discrete/ci_hsc"].load(os.path.join(getPackageDir("ci_hsc_gen2"), "skymap.py"))
     convertRepoConfig.rootSkyMapName = "discrete/ci_hsc"
+    convertRepoConfig.runs["jointcal_photoCalib"] = "preprocessed/jointcal"
+    convertRepoConfig.runs["jointcal_wcs"] = "preprocessed/jointcal"
     butler3 = makeButler(config)
     convertRepoTask = ConvertRepoTask(config=convertRepoConfig, butler3=butler3)
     convertRepoTask.run(
         root=REPO_ROOT,
-        collections=["shared/ci_hsc"],
-        calibs={"CALIB": ["calib/hsc"]},
-        reruns={"rerun/ci_hsc": ["shared/ci_hsc"]},
+        calibs={"CALIB": "calib/hsc"},
+        reruns=[Rerun(path="rerun/ci_hsc", runName="shared/ci_hsc/direct", chainName="shared/ci_hsc",
+                      parents=["shared/ci_hsc/direct"])],
```

Comment:
`parents` is supposed to be used for other Gen2 repos other than the root that are in the Gen2 parent chain.  The Gen3 run created for the rerun is automatically added to the Gen3 chain, so making this a non-empty list just adds it twice, which is harmless, but I'll go ahead and make it an empty list before merging.